File: 17/17_1/17_1.py
k = 0
with open("17_v1_2.txt", 'r') as f:
    min = 2001
    lst = []
    for line in f:
        a = int(line)
        lst.append(a)
    for i in range(len(lst) - 1):
        if(lst[i] % 5 == 0 and lst[i + 1] % 5 == 0):
            k += 1
            s = lst[i] + lst[i + 1]
            if s < min:
                min = s
                
print(k, min)     

File: 17/17_1/17_1.cpp
#include <iostream>
#include <fstream>
#include <vector>
int main() {
	std::fstream f("17_v1_2.txt");
	std::vector<int> v;
	int x, min = 2001, s, count = 0;
	while(!f.eof()) {
		f >> x;
		v.push_back(x);
	}
	
	for(int i = 1; i < v.size() - 1; ++i) {
		if(v[i] % 5 == 0 && v[i + 1] % 5 == 0) {
			s = v[i] + v[i + 1];
			if(s < min)
				min = s;
			count++;
		}
	}
	std::cout << count << ' ' << min;
}
	
